cmd/embed/main.go

```diff
@@ -120,13 +121,25 @@ func run(ctx context.Context) error {
 		}
 	}
 	var err error
+	usedNames := make(map[string]bool)
```

Comment:
Nit: we more commonly use `map[string]bool{}` if you're not going to specify an initial capacity